diff --git a/test/extended/netobserv/loki_storage.go b/test/extended/netobserv/loki_storage.go
index 6d5995161..f097d8f01 100644
--- a/test/extended/netobserv/loki_storage.go
+++ b/test/extended/netobserv/loki_storage.go
@@ -45,15 +45,17 @@ type s3Credential struct {
 }
 
 func getAWSCredentialFromCluster(oc *exutil.CLI) s3Credential {
-	region, err := exutil.GetAWSClusterRegion(oc)
-	o.Expect(err).NotTo(o.HaveOccurred())
+	//region, err := exutil.GetAWSClusterRegion(oc)
+	//o.Expect(err).NotTo(o.HaveOccurred())
+
+	region := "us-east-1"
 
 	dirname := "/tmp/" + oc.Namespace() + "-creds"
 	defer os.RemoveAll(dirname)
-	err = os.MkdirAll(dirname, 0777)
+	err := os.MkdirAll(dirname, 0777)
 	o.Expect(err).NotTo(o.HaveOccurred())
 
-	_, err = oc.AsAdmin().WithoutNamespace().Run("extract").Args("secret/aws-creds", "-n", "kube-system", "--confirm", "--to="+dirname).Output()
+	_, err = oc.AsAdmin().WithoutNamespace().Run("extract").Args("secret/aws-creds", "-n", "default", "--confirm", "--to="+dirname).Output()
 	o.Expect(err).NotTo(o.HaveOccurred())
 
 	accessKeyID, err := os.ReadFile(dirname + "/aws_access_key_id")
@@ -606,11 +608,12 @@ func useExtraObjectStorage(oc *exutil.CLI) string {
 		return "minio"
 	}
 	if strings.Contains(err.Error(), "No resources found") || strings.Contains(err.Error(), "not found") {
-		e2e.Logf("deploy MinIO and use this MinIO as storage service")
-		deployMinIO(oc)
-		return "minio"
+		//e2e.Logf("deploy MinIO and use this MinIO as storage service")
+		//deployMinIO(oc)
+		e2e.Logf("Using AWS S3 as a default object storage ...")
+		return "s3"
 	}
-	return ""
+	return "s3"
 }
 
 func patchLokiOperatorWithAWSRoleArn(oc *exutil.CLI, subNamespace, roleArn string) {
diff --git a/test/extended/netobserv/test_flowcollector.go b/test/extended/netobserv/test_flowcollector.go
index a97b467db..0dd07c6ae 100644
--- a/test/extended/netobserv/test_flowcollector.go
+++ b/test/extended/netobserv/test_flowcollector.go
@@ -350,10 +350,12 @@ var _ = g.Describe("[sig-netobserv] Network_Observability", func() {
 
 			g.By("Deploy lokiStack")
 			// get storageClass Name
-			sc, err := getStorageClassName(oc)
-			if err != nil || len(sc) == 0 {
-				g.Skip("StorageClass not found in cluster, skip this case")
-			}
+			//sc, err := getStorageClassName(oc)
+			//if err != nil || len(sc) == 0 {
+			//	g.Skip("StorageClass not found in cluster, skip this case")
+			//}
+
+			sc := "nfs-storage-provisioner"
 
 			objectStorageType := getStorageType(oc)
 			if len(objectStorageType) == 0 && ipStackType != "ipv6single" {
@@ -364,7 +366,7 @@ var _ = g.Describe("[sig-netobserv] Network_Observability", func() {
 			ls = &lokiStack{
 				Name:          "lokistack",
 				Namespace:     lokiStackNS,
-				TSize:         "1x.demo",
+				TSize:         "1x.extra-small",
 				StorageType:   objectStorageType,
 				StorageSecret: "objectstore-secret",
 				StorageClass:  sc,
--- a/test/extended/testdata/netobserv/kafka/topic-consumer-tls.yaml
+++ b/test/extended/testdata/netobserv/kafka/topic-consumer-tls.yaml
@@ -48,7 +48,7 @@ objects:
           restartPolicy: Never
 parameters:
 - name: KAFKA_IMAGE
-  value: "registry.redhat.io/amq-streams/kafka-34-rhel8:2.5.2"
+  value: "quay.io/openshifttest/kafka@sha256:2411662d89dd5700e1fe49aa8be1219843948da90cfe51a1c7a49bcef9d22dab"
 - name: KAFKA_TOPIC
   value: "network-flows-export"
 - name: KAFKA_USER
diff --git a/test/extended/testdata/netobserv/loki/lokistack-simple.yaml b/test/extended/testdata/netobserv/loki/lokistack-simple.yaml
index 21a6c1cd1..27d346651 100644
--- a/test/extended/testdata/netobserv/loki/lokistack-simple.yaml
+++ b/test/extended/testdata/netobserv/loki/lokistack-simple.yaml
@@ -36,7 +36,7 @@ parameters:
 - name: StorageType
   value: "s3"
 - name: StorageClass
-  value: "gp3-csi"
+  value: "nfs-storage-provisioner"
 - name: Tenant
   value: openshift-network
 - name: EnableIPV6
