--- a/test/extended/netobserv/loki_storage.go
+++ b/test/extended/netobserv/loki_storage.go
@@ -52,15 +52,17 @@ type resource struct {
 }
 
 func getAWSCredentialFromCluster(oc *exutil.CLI) s3Credential {
-	region, err := exutil.GetAWSClusterRegion(oc)
-	o.Expect(err).NotTo(o.HaveOccurred())
+	//region, err := exutil.GetAWSClusterRegion(oc)
+	//o.Expect(err).NotTo(o.HaveOccurred())
+
+	region := "us-east-1"
 
 	dirname := "/tmp/" + oc.Namespace() + "-creds"
 	defer os.RemoveAll(dirname)
-	err = os.MkdirAll(dirname, 0777)
+	err := os.MkdirAll(dirname, 0777)
 	o.Expect(err).NotTo(o.HaveOccurred())
 
-	_, err = oc.AsAdmin().WithoutNamespace().Run("extract").Args("secret/aws-creds", "-n", "kube-system", "--confirm", "--to="+dirname).Output()
+	_, err = oc.AsAdmin().WithoutNamespace().Run("extract").Args("secret/aws-creds", "-n", "default", "--confirm", "--to="+dirname).Output()
 	o.Expect(err).NotTo(o.HaveOccurred())
 
 	accessKeyID, err := os.ReadFile(dirname + "/aws_access_key_id")
@@ -637,7 +639,7 @@ func useExtraObjectStorage(oc *exutil.CLI) string {
 		deployMinIO(oc)
 		return "minio"
 	}
-	return ""
+	return "s3"
 }
 
 // return the storage type per different platform
@@ -662,7 +664,9 @@ func getStorageType(oc *exutil.CLI) string {
 		}
 	default:
 		{
-			return useExtraObjectStorage(oc)
+			//return useExtraObjectStorage(oc)
+			return "s3"
+
 		}
 	}
 }
diff --git a/test/extended/netobserv/test_flowcollector.go b/test/extended/netobserv/test_flowcollector.go
index e4e26e05f..2e858a1b6 100644
--- a/test/extended/netobserv/test_flowcollector.go
+++ b/test/extended/netobserv/test_flowcollector.go
@@ -140,10 +140,11 @@ var _ = g.Describe("[sig-netobserv] Network_Observability", func() {
 
 		g.By("Deploy lokiStack")
 		// get storageClass Name
-		sc, err := getStorageClassName(oc)
-		if err != nil || len(sc) == 0 {
-			g.Skip("StorageClass not found in cluster, skip this case")
-		}
+		//sc, err := getStorageClassName(oc)
+		sc := "nfs-storage-provisioner"
+		//if err != nil || len(sc) == 0 {
+		//	g.Skip("StorageClass not found in cluster, skip this case")
+		//}
 
 		lokiTenant := "openshift-network"
 
@@ -155,7 +156,7 @@ var _ = g.Describe("[sig-netobserv] Network_Observability", func() {
 		ls = &lokiStack{
 			Name:          "lokistack",
 			Namespace:     namespace,
-			TSize:         "1x.demo",
+			TSize:         "1x.extra-small",
 			StorageType:   objectStorageType,
 			StorageSecret: "objectstore-secret",
 			StorageClass:  sc,
@@ -816,8 +817,8 @@ var _ = g.Describe("[sig-netobserv] Network_Observability", func() {
 		o.Expect(err).NotTo(o.HaveOccurred())
 		bearerToken := getSAToken(oc, "netobserv-plugin", namespace)
 
-		g.By("Wait for a min before logs gets collected and written to loki")
-		time.Sleep(60 * time.Second)
+		g.By("Wait for 2 mins before logs gets collected and written to loki")
+		time.Sleep(120 * time.Second)
 
 		// Scenario1: Verify SCTP traffic
 		lokilabels := Lokilabels{
diff --git a/test/extended/testdata/netobserv/kafka/kafka-default.yaml b/test/extended/testdata/netobserv/kafka/kafka-default.yaml
index 04b6bddcc..2e9f4d82e 100644
--- a/test/extended/testdata/netobserv/kafka/kafka-default.yaml
+++ b/test/extended/testdata/netobserv/kafka/kafka-default.yaml
@@ -73,4 +73,4 @@ parameters:
 - name: NAMESPACE
   value: "netobserv"
 - name: STORAGE_CLASS
-  value: "gp2"
+  value: "nfs-storage-provisioner"
diff --git a/test/extended/testdata/netobserv/kafka/topic-consumer-tls.yaml b/test/extended/testdata/netobserv/kafka/topic-consumer-tls.yaml
index b7114d752..7c6f5fb50 100644
--- a/test/extended/testdata/netobserv/kafka/topic-consumer-tls.yaml
+++ b/test/extended/testdata/netobserv/kafka/topic-consumer-tls.yaml
@@ -48,7 +48,7 @@ objects:
           restartPolicy: Never
 parameters:
 - name: KAFKA_IMAGE
-  value: "registry.redhat.io/amq-streams/kafka-34-rhel8:2.5.2"
+  value: "quay.io/openshifttest/kafka@sha256:2411662d89dd5700e1fe49aa8be1219843948da90cfe51a1c7a49bcef9d22dab"
 - name: KAFKA_TOPIC
   value: "network-flows-export"
 - name: KAFKA_USER
diff --git a/test/extended/testdata/netobserv/loki/lokistack-simple.yaml b/test/extended/testdata/netobserv/loki/lokistack-simple.yaml
index 21a6c1cd1..27d346651 100644
--- a/test/extended/testdata/netobserv/loki/lokistack-simple.yaml
+++ b/test/extended/testdata/netobserv/loki/lokistack-simple.yaml
@@ -36,7 +36,7 @@ parameters:
 - name: StorageType
   value: "s3"
 - name: StorageClass
-  value: "gp3-csi"
+  value: "nfs-storage-provisioner"
 - name: Tenant
   value: openshift-network
 - name: EnableIPV6
